frontend/packages/kubevirt-plugin/src/constants/vm/constants.ts

```diff
@@ -43,6 +43,24 @@ export enum DeviceType {
   DISK = 'DISK',
 }
 
+export enum VMWizardActionLabels {
```

Comment:
I know I am picky, but could we move these to `./ui.ts` ?

I guess the hrefs above should go there as well, but le's not do it here now.